src/components/DownloadButton/DownloadButton.js

```diff
@@ -59,14 +60,22 @@ const DownloadButton = (props) => {
   };
 
   return (
-    <Button
+    <DesignSystemButton
       className={className}
-      onClickHandler={onClickDownload}
-      buttonText={buttonText}
-      ButtonIcon={Icon}
+      onClick={onClickDownload}
+      text={buttonText}
+      textAlways
+      icon={Icon ? <Icon /> : null}
+      type={"secondary"}
       {...otherProps}
     />
   );
 };
 
+DownloadButton.propTypes = {
+  buttonText: PropTypes.string,
+  className: PropTypes.string,
+  Icon: PropTypes.func,
+};
```

Comment:
💭 just wondering if there's going to be errors due to the whole `otherProps` bit too and more props being passed in through that mechanism

Reply:
😱 whoops, I intended to add that too

Reply:
100% fine with this. I actually didn't realise PropTypes ignored this use case so one to remember!